src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,22 @@
+const jwtService = require('../services/jwtService.js');
+
+const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    return res.sendStatus(401);
+  }
+
+  const userData = jwtService.verify(token);
```

Comment:
Wrap the `jwtService.verify(token)` call in a try-catch block to handle any potential errors during token verification. This will prevent unhandled exceptions and allow you to send an appropriate error response.